webapi/src/test/scala/org/knora/webapi/testcontainers/SipiTestContainer.scala

```diff
@@ -45,6 +46,11 @@ object SipiTestContainer {
       "/sipi/config/sipi.docker-config.lua",
       BindMode.READ_ONLY
     )
+
+    // TODO: add some default images: https://www.testcontainers.org/features/files/
+    val incunabulaImageDirPath = Paths.get("..", "sipi/images/0803/incunabula_0000000002.jp2")
+    sipiContainer.withFileSystemBind(incunabulaImageDirPath.toString(), "/sipi/images/0803/incunabula_0000000002.jp2", BindMode.READ_ONLY)
```

Comment:
This could be val `sipi/images/0803/incunabula_0000000002.jp2`.
Also is above TODO fulfilled with these 2 lines?

Reply:
removed the comment. the val cannot be simplified, because the value is not the same then.